src/js/components/Anchor/README.md

```diff
@@ -1,6 +1,8 @@
 ## Anchor
 A text link.
-###### We have a separate component from the browser base so we can style it. You can either set the icon and/or label properties or just use children.
+We have a separate component from the browser
```

Comment:
I think we need an extra newline before this paragraph. As it stands, I think it will be right against the description.